src/java/test/org/apache/zookeeper/server/quorum/LocalPeerBeanTest.java

```diff
@@ -79,4 +80,27 @@ public void testClientAddress() throws Exception {
         cnxnFactory.shutdown();
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testIsLeader() throws Exception {
```

Comment:
- This is a `QuorumPeer` test, move it to `QuorumPeerTest` class please.
- These are actually 3 tests together, split them into 3 methods please.